index.bs

```diff
@@ -1696,22 +1696,78 @@ Team Representative in an Interest Group</h5>
-	feedback on the <a href="#ACCommunication">Advisory Committee discussion list</a>
-	or via other designated channels.
-
-	W3C <em class="rfc2119">may</em> begin work
-	on a Working Group or Interest Group charter
-	at any time.
+	W3C creates [=charters=] for [=chartered groups=] based on interest from the [=Members=] and [=Team=].
+	[=Charters=] are formally approved through an [=AC Review=] and [=W3C Decision=] (see [[#CharterReview]]).
+	Prior to [=AC Review=],
+	<dfn>charter drafts</dfn> go through a public [=charter refinement=] phase
+	during which the [=charter=] receives [=wide review=]
+	and the [=Chartering Facilitator=] attempts to find [=consensus=] on the charter.
+
+<h3 id="charter-initiation">
+Initiating Charter Refinement</h3>
+
+	[=Charter refinement=] (see below) is initiated
+	by the [=Team=] sending an [=charter/Advance Review Notice=]
+	to the [=Advisory Committee=],
+	to the public,
+	and, in the case of rechartering, to the affected [=Group=].
+
+	This <dfn for=charter>Advance Review Notice</dfn> must indicate:
+	* A short summary of the proposal.
+	* The location of the [=charter draft=], which must be public.
+	* How to participate in the discussion of this [=charter draft=] and where to file issues.
+	* The expected duration of the [=charter refinement=] phase,
+		which must not be less than 28 days,
+		and should not be more than 6 months.
+	* Who the [=Chartering Facilitator=] is.
+
+	The [=Team=] is responsible for initiating [=charter refinement=]
+	at its discretion, in consideration of discussions with the community.
+	An [=Advisory Committee representative=] <em class=rfc2119>may</em> formally request
+	that the [=Team=] initiate [=charter refinement=].
+	The Team <em class=rfc2119>may</em> deny such a request
+	if it thinks the proposal is insufficiently mature or does not align with W3C's scope and mission.
+	This rejection is a [=Team Decision=] which can be <a href="#registering-objections">Formally Objected to</a>.
+
+<h3 id="charter-development" oldids="WGCharterDevelopment">
+Charter Refinement</h3>
+
+	During <dfn>charter refinement</dfn>,
+	the W3C community further develops the [=charter draft=]
+	with the goal of achieving [=consensus=] on the proposal.
+	This process is chaired by the <dfn>Chartering Facilitator</dfn>,
+	who acts as [=Chair=] for [=charter refinement=].
+
+	Note: The [=Chartering Facilitator=] is not necessarily (and often is not) the [=Chair=] of the [=chartered group|group=] being chartered.
+
+	During [=charter refinement=]:
+	* All issues filed against the [=charter draft=] must be [=formally addressed=],
+		and their resolutions tracked in a disposition of comments.
+	* [=Wide review=] of the [=charter draft=] is initiated and completed.
+	* Decisions are made as  [=group decisions=] of all individuals participating in this process.
+		(However note that only W3C [=Members=] and the [=Team=] can particpate in a formal [[#Votes|vote]].)
```

Comment:
```suggestion
	* Decisions are made as [=group decisions=] by all individuals participating in this process
		(though only W3C [=Members=] and the [=Team=] can particpate in a formal [[#Votes|vote]]).
```

Reply:
I think I agree with the change on the second line (though I'd be curious about @fantasai 's take).

I don't think "group decisions by all individuals" is right: group decisions are made _by the chair_, in consideration of the consensus of the group. Any other idea on how to make that phrasing sound better without implying that the decision is made differently from the way it normally is?

Reply:
I've almost never understood a group decision to have been *made by the chair.* Rather, the chair almost always _recognizes and formalizes_ decisions *made by the group,* commonly by consensus, sometimes by vote. Only in the _very_ rare occasions that the group could not come to a decision has a chair stepped in to make a decision.

Reply:
The process says:
> decisions taken by the [Chair](https://www.w3.org/2023/Process-20231103/#GeneralChairs) of a [Working Group](https://www.w3.org/2023/Process-20231103/#GroupsWG) or [Interest Group](https://www.w3.org/2023/Process-20231103/#GroupsIG) on the basis of having assessed the [consensus](https://www.w3.org/2023/Process-20231103/#def-Consensus) of the group or following a vote (see [§ 5.2.3 Deciding by Vote](https://www.w3.org/2023/Process-20231103/#Votes)) are called group decisions (also known as group “resolutions”).

Reply:
I'm at a loss as to why my suggestions here have been marked "resolved", as they have not been applied, and the conversations didn't end with either me or the CG saying "drop it". Here's a possibly more palatable minor revision of my previous suggestion *here*, including fixing a typo I overlooked before.
```suggestion
	* Decisions are made as [=group decisions=], where the group is made up of all individuals participating in this process,
		(though only W3C [=Members=] and the [=Team=] can participate in any formal [[#Votes|vote]]).
```

Reply:
Closing this was accidental, and I'm accepting in this latest suggestion by @TallTed as an editorial improvement.